Pod/Classes/AppViewController.swift

```diff
@@ -73,6 +73,11 @@ open class AppViewController: UIViewController {
         toViewController.view.frame = view.bounds
         view.addSubview(toViewController.view)
         
+        toViewController.view.leadingAnchor.constraint(equalTo: view.leadingAnchor).isActive = true
```

Comment:
This is extremely nitpick, but my convention when writing constraints like this are to have them ordered as: top, leading, bottom, trailing.